schema-engine/sql-introspection-tests/tests/commenting_out/mysql.rs

```diff
@@ -147,3 +147,48 @@ PARTITIONS 2; "#,
 
     Ok(())
 }
+
+#[test_connector(tags(Mysql8), exclude(Vitess))]
+async fn mysql_multi_row_index_warning(api: &mut TestApi) -> TestResult {
+    api.raw_cmd(
+        r#"
+CREATE TABLE customers (
+    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    modified DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    custinfo JSON,
+    INDEX zips( (CAST(custinfo->'$.zipcode' AS UNSIGNED ARRAY)) )
```

Comment:
could we also test an index with multiple columns, and a multi-value unique?

Reply:
Added

Reply:
Is it pushed? I think it's not there.

Reply:
I had added such a test in [this commit](https://github.com/prisma/prisma-engines/pull/3867/commits/3fc293250116849807657e9ccaf835eec90e3ee6), and polished it later. Did you mean something else?

Reply:
This is still an index with a single, "multi-value"/expression column.

I meant something like (using the schema in the existing test):

```sql
            UNIQUE comp(modified, (CAST(custinfo->'$.zipcode' AS UNSIGNED ARRAY)))
```

i.e. indexing / constraining a regular column first, then a multi-value virtual column.

There's an example in the [mysql docs](https://dev.mysql.com/doc/refman/8.0/en/create-index.html#create-index-multi-valued).

Reply:
I think we need this test because looking at the logic in this PR, I expect it may not work as intended.

Reply:
I have added this in the test (table: `customers_2`), and it looks like `UNIQUE comp(..)` indeed does not appear at all in the introspected schema